scripts/externalROMloader.py

```diff
@@ -120,6 +120,21 @@ def __setstate__(self, d):
     self._frameworkLoc = d['whereFrameworkIs']
     self._load(self._binaryLoc, self._frameworkLoc)
 
+  def setAdditionalParams(self, nodes):
+    '''
+    Set ROM parameters for external evaluation
+    @ In, nodes, list, list of xml nodes for setting parameters of pickleROM
+    @ Out, None
+    '''
+    from ravenframework.SupervisedLearning.pickledROM import pickledROM
+    spec = pickledROM.getInputSpecification()()
```

Comment:
I'm not sure, but couldn't you just use `self.rom.getInputSpecification()` to get the input specs, or do you need the pickledROM input specs?

Reply:
I believe it raises an error `AttributeError: 'ravenROMexternal' object has no attribute 'getInputSpecification'`